deploy/rosa-console-branding-configmap/rosa-brand-logo.yaml

```diff
@@ -1,6 +1,48 @@
+  rosa-brand-logo.svg: "<?xml version=\"1.0\" encoding=\"utf-8\"?>\n<!-- Generator:
```

Comment:
So hey, no idea why this delta is here, but it was generated when I ran `make` so...

Reply:
@cblecker any idea?

Reply:
Delta was tested on two clusters.

Reply:
I expect the newlines are irrelevant, since they're embedded in XML.

I thought CI would have complained about this if it was a problem... but that job is [only checking the generated deployment yaml](https://github.com/openshift/managed-cluster-config/blob/4d4bb85e9dd4113a2c301633eb7fdb0ad56663ac/hack/app_sre_pr_check.sh#L27-L40).

I'll spin up a new stage cluster once this lands to make sure it's copacetic.